packages/fixie/src/isomorphic-client.ts

```diff
@@ -125,6 +125,8 @@ export class IsomorphicFixieClient {
   addCorpusSource(
     corpusId: string,
     startUrls: string[],
+    includeGlobs?: string[],
+    excludeGlobs?: string[],
```

Comment:
This was a breaking change, and should have been accompanied by a bump to `2.0.0`.

Reply:
Oops, sorry. What about all of the other unintentional breaking changes? :-)

Reply:
😅